dll/win32/framedyn/chstring.cpp

```diff
@@ -348,30 +352,32 @@ void CHString::AllocCopy(CHString& dest, int nCopyLen, int nCopyIndex, int nExtr
     // copying data from another. This is needed by Left/Mid/Right
     dest.AllocBuffer(nCopyLen + nExtraLen);
     // And copy our stuff in
-    wcsncpy(dest.m_pchData, m_pchData + nCopyIndex, nCopyLen);
+    wcsncpy(reinterpret_cast<LPWSTR>(dest.m_pchData),
+            reinterpret_cast<LPWSTR>(m_pchData + nCopyIndex),
+            nCopyLen);
 }
 
 /*
  * @implemented
  */
-BSTR CHString::AllocSysString() const
+CHSTRING_LPWSTR CHString::AllocSysString() const
```

Comment:
Why CHSTRING_LPWSTR? Is it equivalent to BSTR? (Note that https://docs.microsoft.com/en-us/windows/win32/api/chstring/nf-chstring-chstring-allocsysstring says it returns a BSTR.)

Reply:
Mangled export says it returns a pointer to unsigned to unsigned short. So be it.